src/Language/TypeScript/Resolution.hs

```diff
@@ -110,7 +110,6 @@ resolveModule :: ( Has (Modules address value) sig m
               -> [String] -- ^ File extensions to look for
               -> Evaluator term address value m (Either [M.ModulePath] M.ModulePath)
 resolveModule path' exts = do
-  -- JZ:TODO: Refactor later
   let path = makeRelative "." $ Path.toString path'
```

Comment:
This is also related to https://github.com/github/semantic/issues/537.  I will refactor it in another PR.